src/ui/prompts/featurePrompts.ts

```diff
@@ -18,7 +18,7 @@ export const featureChoices = async (input: Record<string, any>):Promise<Feature
 }
 
 export const featurePrompt = {
-    name: '_feature',
```

Comment:
is there a reason for renaming this? might have to fix it in `variations/get` and `variations/list` since the featurePrompt is being used there too

Reply:
ahh, will fix that. it is because I am matching the flag name to the prompt key